plutus-metatheory/src/Parsers.hs

```diff
@@ -0,0 +1,154 @@
+{-# LANGUAGE LambdaCase #-}
```

Comment:
I think this file is identical to the one in `plutus-core/executables`, is that right?  That must mean that it's got lots of stuff that the metatheory doesn't need, but there's probably not much point in removing it since that'll make it difficult to maintain.  Can we re-use the `plutus-core` version directly, or would that require `plutus-core` to export something that it doesn't want to?

Reply:
... and the same comment for `Common.hs`.

Reply:
Yes, the files are copied from `plutus-core/executables`. In [PLT-1312](https://input-output.atlassian.net/browse/PLT-1312) I'll make the original files available as a library and remove the copied files.